### PR TITLE
Bump phabricator, postgreSQL, ghost and mongoDB chart versions

### PR DESCRIPTION
**Pull request by dgonzalezruiz**

### CODE REVIEW

**Comment by chipironcin** [comment]
I cannot find `2016-41` in DockerHub. Also I guess you should bump the chart version too in Chart.yaml

**Comment by sameersbn** [comment]
even https://github.com/bitnami/bitnami-docker-phabricator/blob/master/Dockerfile indicates that the last released version is `2016.40-r0`.

**Comment by dgonzalezruiz** [comment]
Updated the PR. Sorry about the confussion generated!

File: ghost/values.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Bitnami Ghost image version
 ## ref: https://hub.docker.com/r/bitnami/ghost/tags/
 ##
-imageTag: 0.11.0-r1
+imageTag: 0.12.0-r0
```
Comment (sameersbn) [inline ghost/values.yaml]: this tag does on exist on docker hub

**Comment by sameersbn** [comment]
closing, since the charts this PR addresses have been updated in separate PR's

**Pull request closed without merging**